inc/classes/class-plugin.php

```diff
@@ -73,6 +73,8 @@ public function load() {
 		$event_manager = new Event_Manager();
 		$subscribers   = [];
 
+		$event_manager->add_subscriber( new Plugins\Mobile_Subscriber() );
```

Comment:
This one is added directly because it must be done BEFORE the line `new Settings_Page( ... )`, otherwise the filters on WPR settings wouldn't work.